binaries/http/main.go

```diff
@@ -21,13 +21,15 @@ func main() {
 		os.Exit(1)
 	}
 
-	executedRequestResponse, executeRequestError := daemon.ExecuteRequest(options)
+	executedRequestResponses, executeRequestError := daemon.ExecuteRequest(options)
```

Comment:
Nit: the `execute` prefix seems redundant? Why not `requestResponses` and `requestError`? It feels like the fact that the values were executed is implied?

Reply:
👍